test/extended/cli/version.go

```diff
@@ -0,0 +1,101 @@
+package cli
+
+import (
+        "encoding/json"
+        "fmt"
+        "log"
+        "os"
+        "regexp"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+)
+
+var _ = g.Describe("[cli]oc version[Conformance]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc                     = exutil.NewCLI("oc", exutil.KubeConfigPath())
+	)
+
+	g.Describe("oc version before login", func() {
+		g.It("check version without login", func() {
+			g.By("check version info with default KUBECONFIG") 
+			out, err := oc.Run("version").Args("-o", "json").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+                        versionInfo := &VersionInfo{}
+                        if err := json.Unmarshal([]byte(out), &versionInfo); err != nil {
+                                log.Fatal("unable to decode version with error: %v", err)
+                        }
+                        reg := regexp.MustCompile(`(openshift-clients-)?[\d]+\.[\d]+\.[\d]+\-[\d]{12}(\-[\w]+)?`)
+                        if match := reg.MatchString(versionInfo.ClientInfo.GitVersion); !match {
+                                log.Fatal("varification version with error: %v", err) 
+                        }         
+
+			g.By("check version info without --kubeconfig,~/.kube/config,KUBECONFIG") 
+                        kubeconfig :=exutil.KubeConfigPath()
```

Comment:
You can still default to `$HOME/.kube/config` so you need to ensure one does not exist.

Reply:
@soltysh  I've updated , please help review again, thanks.